src/components/templates/Camp/components/Map.scss

```diff
@@ -129,6 +131,145 @@ $border-radius: 28px;
     opacity: 1;
     transform: translateY(2px) scale(1);
   }
+
+  .camp-venue {
+    display: flex;
+    justify-content: center;
+    z-index: 1;
+    width: 100%;
+    height: 100%;
+    cursor: pointer;
+    &:hover {
+      .camp-venue-text {
+        display: block;
+      }
+      .camp-venue-img {
+        transform: scale(1.05);
+      }
+    }
+
+    &.clicked {
+      .camp-venue-text {
+        .camp-venue-secondinfo {
+          display: block;
+        }
+      }
+    }
+    &:active {
+      .camp-venue-img {
+        transform: scale(1);
+      }
+    }
+    .camp-venue-img {
+      position: relative;
+      width: 100%;
+      height: 100%;
+      transition: all 400ms cubic-bezier(0.23, 1, 0.32, 1);
```

Comment:
This is a nitpick:

When adding transition, try not to add the _all_, that basically means that the browser will watch all transitionable properties and apply the timing to them.

Rather, put the specific property that you want to change i.e.

```
transition:
    width 400ms cubic-bezier(0.23, 1, 0.32, 1),
    height 400ms cubic-bezier(0.23, 1, 0.32, 1)
```

Reply:
TIL!